settings.py

```diff
@@ -395,7 +396,18 @@
     SERVICE_ACCOUNT_FOR_ANVIL = subprocess.run(['gcloud auth list --filter=status:ACTIVE --format="value(account)"'],
                                                capture_output=True, text=True, shell=True).stdout.split('\n')[0]
     if not SERVICE_ACCOUNT_FOR_ANVIL:
-        raise Exception('Error starting seqr - gcloud auth is not properly configured')
+        # attempt to acquire a service account token
+        if os.path.exists('/.config/service-account-key.json'):
+            SERVICE_ACCOUNT_FOR_ANVIL = \
+                re.findall('\[(.*)\]',
+                           subprocess.run(['gcloud', 'auth', 'activate-service-account', '--key-file',
+                                           '/.config/service-account-key.json'],
+                                           capture_output=True, text=True).stderr)[0] # nosec
```

Comment:
i had to add the `# nosec` here to pass codacy. It initially complained about the use of `shell=True`, which i thought was reasonable. But, after doing what I thought it wanted, and using a `shell=False`, it complained about unsanitized command input... but the input here is a static list. Let me know if you have any thoughts on this (or thoughts on how to improve the logic in this section in general, @hanars )